XCUITests/PerformanceTestPlan.xctestplan

```diff
@@ -0,0 +1,76 @@
+{
+  "configurations" : [
+    {
+      "id" : "A98C0147-1E50-47C8-8214-6E811BD56870",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+
+  },
+  "testTargets" : [
```

Comment:
How do these get updated in testplans when tests are added or removed?

Reply:
that happens automatically when you select the test plan from xcode and add/remove tests